tests/scripts/list-macros.sh

```diff
@@ -8,6 +8,8 @@ if [ -d include/mbedtls ]; then :; else
 fi
 
 HEADERS=$( ls include/mbedtls/*.h crypto/include/mbedtls/*.h | egrep -v 'compat-1\.3\.h' )
+HEADERS="$HEADERS
+$( ls library/*.h )"
```

Comment:
On the same line?

Reply:
No, it's deliberately on a separate line since the list is newline-separated.